tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "x-pack/plugins/cases/**/*",
     "x-pack/plugins/lists/**/*",
     "x-pack/plugins/security_solution/**/*",
+    "x-pack/plugins/metrics_entities/**/*",
```

Comment:
I think the new pattern here is to leverage `references` below instead of `include`.

Reply:
I will change these out with this PR and see if it works.